.github/actions/external_runner/main.go

```diff
@@ -74,7 +74,21 @@ func LoadConfig() Config {
 
 func main() {
 	params := LoadConfig()
+	taskRunner, cfg := prepareFargateTask(params)
 
+	id, err := runFargateTask(time.Duration(params.TimeoutMillis)*time.Millisecond, taskRunner)
```

Comment:
maybe extract the timeout into a variable and remove the empty line before error checking?